frontend/components/job-details.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div v-if="false" class="panel panel-default">
+    <div class="panel-heading">
+      <button type="button" class="close" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+      <h3 class="panel-title">{{job.name}}</h3>
+      <template v-if="repeating"><span class="label label-info"><i class="glyphicon glyphicon-repeat"></i> {{ job.repeatInterval }}</span></template>
```

Comment:
Instead of `glyphicon` we could probably just use this: ↻

Looks like this is the only icon in the project. :-)

Reply:
I'll enable tree shaking later so it'll just bundle the single `glyphicon`. That'll allow us to use them more later on if we choose.

Reply:
great!